src/Uno.Toolkit.UI/Behaviors/FlipView.xaml

```diff
@@ -0,0 +1,34 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+	<Style x:Key="FlipViewNoArrowsStyle" TargetType="FlipView">
```

Comment:
```suggestion
	<Style x:Key="NoArrowsFlipViewStyle" TargetType="FlipView">
```
Typically the convention is "[qualifiers...}{control-type-name}Style", with few exceptions (eg: textblock styles that needs to match material design).